src/db/collections/outputs.rs

```diff
@@ -14,30 +14,38 @@ use crate::{
     db::MongoDb,
     types::{
         ledger::{MilestoneIndexTimestamp, OutputMetadata, OutputWithMetadata, SpentMetadata},
-        stardust::block::{BlockId, Output, OutputId},
+        stardust::block::{Address, BlockId, Output, OutputId},
         tangle::MilestoneIndex,
     },
 };
 
 /// Chronicle Output record.
 #[derive(Clone, Debug, Serialize, Deserialize)]
-struct OutputDocument {
+pub(crate) struct OutputDocument {
     output_id: OutputId,
     output: Output,
     metadata: OutputMetadata,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    address: Option<Address>,
+    is_trivial_unlock: bool,
```

Comment:
I wonder if we should create a new struct for values that we have precomputed? I'd like to keep these things separated as much as possible.

Reply:
But then you have to come up with a name!! :trollface:

Reply:
How about any of these: `Output{PreCalculated|Latent}{Details|Information}`?

Reply:
alright, thanks. I'll throw a die on each of the choices.